target/linux/kirkwood/files/arch/arm/boot/dts/kirkwood-lenovo-ix2-common.dtsi

```diff
@@ -0,0 +1,221 @@
+ *
+ * Copyright (c) 2015-2016 bodhi <mibodhi@gmail.com>
+ * Copyright (c) 2015      balbes150
+ *
+ * This file is licensed under the terms of the GNU General Public
+ * License version 2.  This program is licensed "as is" without any
+ * warranty of any kind, whether express or implied.
+ */
+
+/dts-v1/;
+#include "kirkwood.dtsi"
+#include "kirkwood-6282.dtsi"
+
+/ {
+	/* 256 MB RAM. */
+	memory {
+		device_type = "memory";
+		reg = <0x00000000 0x10000000>;
+	};
+
+	chosen {
+		bootargs = "console=ttyS0,115200n8 earlyprintk";
+		stdout-path = &uart1;
+	};
+
+	ocp@f1000000 {
+		pinctrl: pin-controller@10000 {
+
+			pmx_power_off: pmx-power-off {
+				marvell,pins = "mpp36";
+				marvell,function = "gpio";
+			};
+
+			pmx_sata1_power: pmx-sata1-power {
+				marvell,pins = "mpp17";
+				marvell,function = "gpio";
+			};
+
+			pmx_btn_power: pmx-btn-power {
+				marvell,pins = "mpp15";
+				marvell,function = "gpio";
+			};
+
+			pmx_btn_reset: pmx-btn-reset {
+				marvell,pins = "mpp29";
+				marvell,function = "gpio";
+			};
+
+			pmx_led_sys_white: pmx-led-sys-white {
+				marvell,pins = "mpp39";
+				marvell,function = "gpio";
+			};
+
+			pmx_led_sys_red: pmx-led-sys-red {
+				marvell,pins = "mpp37";
+				marvell,function = "gpio";
+			};
+
+			pmx_led_hdd_blue: pmx-led-hdd-blue {
+				marvell,pins = "mpp26";
+				marvell,function = "gpio";
+			};
+
+			pmx_led_hdd1_red: pmx-led-hdd1-red {
+				marvell,pins = "mpp25";
+				marvell,function = "gpio";
+			};
+
+			pmx_led_hdd2_red: pmx-led-hdd2-red {
+				marvell,pins = "mpp24";
+				marvell,function = "gpio";
+			};
+
+			pmx_led_brightness: pmx-led-brightness {
+				marvell,pins = "mpp40", "mpp41";
+				marvell,function = "gpio";
+			};
+		};
+
+		/* Second serial port is available on header "CN4". */
+		serial@12100 {
+			status = "okay";
+		};
+
+		rtc@10300 {
+			status = "okay";
+		};
+
+		i2c@11000 {
+			status = "okay";
+
+			adt7473: adt7473@2e {
+				compatible = "adi,adt7473";
+				reg = <0x2e>;
+			};
+
+			/* Also something at 0x64. Maybe Marvell i2c debug? */
+		};
+
+		sata@80000 {
+			status = "okay";
+			nr-ports = <2>;
+		};
+	};
+
+	gpio-poweroff {
+		compatible = "gpio-poweroff";
+		pinctrl-0 = <&pmx_power_off>;
+		pinctrl-names = "default";
+		gpios = <&gpio1 4 GPIO_ACTIVE_HIGH>;
+	};
+
+	gpio-keys {
+		compatible = "gpio-keys";
+		#address-cells = <1>;
+		#size-cells = <0>;
+		pinctrl-0 = <&pmx_btn_power &pmx_btn_reset>;
+		pinctrl-names = "default";
+
+		power {
+			label = "Power Button";
+			linux,code = <KEY_POWER>;
+			gpios = <&gpio0 15 GPIO_ACTIVE_HIGH>;
+		};
+
+		reset {
+			label = "Reset Button";
+			linux,code = <KEY_RESTART>;
+			gpios = <&gpio0 29 GPIO_ACTIVE_LOW>;
+		};
+	};
+
+	gpio-leds {
+		compatible = "gpio-leds";
+		pinctrl-0 = <&pmx_led_sys_white &pmx_led_sys_red
+			     &pmx_led_hdd_blue
+			     &pmx_led_hdd1_red &pmx_led_hdd2_red>;
+		pinctrl-names = "default";
+
+		white-sys {
+			label = "ix2:white:sys";
```

Comment:
please remove model prefix, just "white:sys", "red:sys", etc.

Why "sys" and not "system"?